### ISSUE TITLE
AccTest: auto scaling policy test failure

### ISSUE THREAD

**Issue by jungchanSon** [NaverCloudPlatform/terraform-provider-ncloud#351]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform Ncloud Provider Version

```text
Terraform version : 1.5.4
Ncloud Provider version : 2.3.18
```

### Affected Resource(s)

* auto_scaling_policy

### Terraform Configuration Files

Please include all Terraform configurations required to reproduce the bug. Bug reports without a functional reproduction may be closed without investigation.

```hcl
none
```

### Panic Output
```
--- FAIL: TestAccResourceNcloudAutoScalingPolicy_classic_basic (392.71s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x8 pc=0x11dbcb5]

goroutine 50 [running]:
testing.tRunner.func1.2({0x14645a0, 0x1eb9940})
        C:/Program Files/Go/src/testing/testing.go:1396 +0x24e
testing.tRunner.func1()
        C:/Program Files/Go/src/testing/testing.go:1399 +0x39f
panic({0x14645a0, 0x1eb9940})
        C:/Program Files/Go/src/runtime/panic.go:884 +0x212
github.com/terraform-providers/terraform-provider-ncloud/internal/service/autoscaling.getClassicAutoScalingPolicy(0xc000552140, {0xc000576180, 0x21}, {0xc00050e5e0?, 0xc0003327e0?})

        C:/Users/x/Desktop/terraform_naver/terraform-provider-ncloud/internal/service/autoscaling/auto_scaling_policy.go:196 +0xf5
github.com/terraform-providers/terraform-provider-ncloud/internal/service/autoscaling.GetAutoScalingPolicy(0x1458fc0?, {0xc000576180?, 0x15db133?}, {0xc00050e5e0?, 0x0?})
        C:/Users/x/Desktop/terraform_naver/terraform-provider-ncloud/internal/service/autoscaling/auto_scaling_policy.go:152 +0x38
github.com/terraform-providers/terraform-provider-ncloud/internal/service/autoscaling_test.testAccCheckNcloudAutoScalingPolicyDestroy(0x0?, 0x0?)
        C:/Users/x/Desktop/terraform_naver/terraform-provider-ncloud/internal/service/autoscaling/auto_scaling_policy_test.go:322 +0x13a
github.com/terraform-providers/terraform-provider-ncloud/internal/service/autoscaling_test.TestAccResourceNcloudAutoScalingPolicy_classic_basic.func2(0x17c1c40?)
        C:/Users/x/Desktop/terraform_naver/terraform-provider-ncloud/internal/service/autoscaling/auto_scaling_policy_test.go:28 +0x25
github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource.runPostTestDestroy({0x17c1c40, 0xc000632180}, {0x17ca3a0, 0xc000507ba0}, {0x0, 0xc00030a690, 0x0, 0xc000382fc0, 0x0, 0x0
, ...}, ...)
        C:/Users/x/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.26.1/helper/resource/testing_new.go:32 +0x150
github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource.runNewTest.func1()
        C:/Users/x/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.26.1/helper/resource/testing_new.go:76 +0x394
runtime.Goexit()
        C:/Program Files/Go/src/runtime/panic.go:522 +0x185
testing.(*common).FailNow(0xc000507ba0)
        C:/Program Files/Go/src/testing/testing.go:871 +0x4c
testing.(*common).Fatalf(0xc000507ba0, {0x15d9826?, 0x15d3b60?}, {0xc0009c28d0?, 0x15d492c?, 0x10?})
        C:/Program Files/Go/src/testing/testing.go:955 +0x65
github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource.runNewTest({0x17c1c40, 0xc000632180}, {0x17ca3a0, 0xc000507ba0}, {0x0, 0xc00030a690, 0x0, 0xc000382fc0, 0x0, 0x0, ...},
...)
        C:/Users/x/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.26.1/helper/resource/testing_new.go:314 +0x166b
github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource.Test({0x17ca3a0, 0xc000507ba0}, {0x0, 0xc00030a690, 0x0, 0xc000382fc0, 0x0, 0x0, 0x0, 0x0, ...})
        C:/Users/x/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.26.1/helper/resource/testing.go:776 +0x64f
github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource.ParallelTest({0x17ca3a0, 0xc000507ba0}, {0x0, 0xc00030a690, 0x0, 0xc000382fc0, 0x0, 0x0, 0x0, 0x0, ...})
        C:/Users/x/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.26.1/helper/resource/testing.go:679 +0x88
github.com/terraform-providers/terraform-provider-ncloud/internal/service/autoscaling_test.TestAccResourceNcloudAutoScalingPolicy_classic_basic(0xc000507ba0)
        C:/Users/x/Desktop/terraform_naver/terraform-provider-ncloud/internal/service/autoscaling/auto_scaling_policy_test.go:24 +0x6a8
testing.tRunner(0xc000507ba0, 0x1648d78)
        C:/Program Files/Go/src/testing/testing.go:1446 +0x10b
created by testing.(*T).Run
        C:/Program Files/Go/src/testing/testing.go:1493 +0x35f
FAIL    github.com/terraform-providers/terraform-provider-ncloud/internal/service/autoscaling   393.343s
```

### Expected Behavior
Pass the test

### Actual Behavior
Fail the test

### Steps to Reproduce
```
TF_ACC=1 go test ./internal/service/autoscaling/... -run TestAccResourceNcloudAutoScalingPolicy_classic_basic -v
```

**Comment by jungchanSon** [comment]
i am working on this

**Comment by wonchulee** [comment]
resolved by #352
